lib/game.rb

```diff
@@ -1,66 +1,133 @@
 class Game 
   attr_reader :player_1,
-              :player_2
+              :player_2,
+              :board,
+              :column_choices
 
   def initialize
-    @player_1 = ""
-    @player_2 = ""
+    @board = Board.new
+    @is_bot = false
+    @column_choices = ["A", "B", "C", "D", "E", "F", "G"]
+    @player_1_turn = ""
+    @player_2_turn = ""
+    @player_1_selection = ""
+    @turn_count = 1
+  end
+
+  def user_input
+    gets.chomp
```

Comment:
this can be changed to `gets.chomp.upcase` to address the case sensitivity issue you had mentioned in the PR

Reply:
Cool. Would you like to make that edit or would you like me to make it?

Reply:
I can take care of this one